_docs_use-cases/use-case-consumption-get-shared-versions-of-an-attribute.md

```diff
@@ -1,6 +1,8 @@
 ---
 # Start automatic generation
 permalink: use-case-consumption-get-shared-versions-of-an-attribute
+redirect_from:
```

Comment:
Please add this in our Excel table as well. (Should only be done shortly before merging v5 into main.)